dev-tools/audius-compose

```diff
@@ -89,6 +90,12 @@ def generate_env(
     env["BOOTSTRAP_SP_OWNER_WALLETS"] = ",".join(bootstrap_sp_owner_wallets)
     env["BOOTSTRAP_SP_DELEGATE_WALLETS"] = ",".join(bootstrap_sp_owner_wallets)
 
+    aao_wallets = []
+    for key, value in env.items():
+        if key.startswith("AAO_WALLET_") and key.endswith("_ADDRESS"):
```

Comment:
curious why not `key == 'AAO_WALLET_ADDRESS'`

Reply:
The code is written this way to allow registering multiple AAO addresses which can be useful for testing

Reply:
cool